packages/app/src/specs/SpecsListRowItem.vue

```diff
@@ -20,28 +20,54 @@
       >
         <div
           data-cy="specs-list-row-git-info"
-          class="col-span-2"
         >
           <slot name="git-info" />
         </div>
-        <div>
-          <slot name="latest-runs" />
-        </div>
-        <div class="hidden md:col-span-2 md:block">
-          <slot name="average-duration" />
-        </div>
+        <SpecsListHoverCell
+          data-cy="specs-list-row-latest-runs"
+          :is-hover-disabled="hasRuns"
+        >
+          <template #content>
+            <slot name="latest-runs" />
+          </template>
+          <template #hover>
+            <slot
+              name="connect-button"
+              :utmMedium="'Specs Latest Runs Empty State'"
```

Comment:
Since it's a string that won't change we can do 
```suggestion
              utmMedium="Specs Latest Runs Empty State"
```

Vue can optimize this more now, as well, since it's never going to change.

Reply:
Fixed